src/index.js

```diff
@@ -1,9 +1,9 @@
-import 'Globals';
+import 'reactPolyfills';
+import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import 'Objs/index';
-import 'Widgets/index';
+import 'Objs/';
```

Comment:
FYI not sure about what the spec says, but my experience is that the trailing slash is not needed.